docs/wiki/Whats-new.md

```diff
@@ -73,6 +73,17 @@ Here's what's changed in Enterprise Scale/Azure Landing Zones:
     - [Deny vNet peering cross subscription.](https://www.azadvertizer.net/azpolicyadvertizer/Deny-VNET-Peer-Cross-Sub.html)
     - Portal accelerator updated with additional tab and options to enable this initiative.
 - Added assignment of "Network interfaces should not have public IPs" built-in Policy to the 'Corp' Management Group.
+- Added a policy assignment to deny Classic resources at the intermediate root management group
+- Added new initiative and assignment to implement recommended guardrails for Azure Key Vault at the landing zones management group
+  - Initiative name: ENFORCE-Guardrails-KeyVault
+  - Policies included: [ALZ Polices](https://aka.ms/alz/policies)
+  - Portal accelerator updated
+- Added two new policy assignments to govern Corp Management Group networking:
+  - DENY-AlzCorpNetworking - blocks the provisioning onf vWAN/ER/VPN gateways in Corp
```

Comment:
Should we rename this to something more specific. eg. `Deny-HybridNetworking` as its a bit generic ATM and could lead to "panic" by consumers not understanding it without digging into it